### PR TITLE
Add support phpStorm 2023.3 EAP

### PR DESCRIPTION
**Pull request by CrazyBoy49z**
https://plugins.jetbrains.com/docs/intellij/tools-intellij-platform-gradle-plugin-extension.html#intellijPlatform-pluginConfiguration-ideaVersion

#1  Maybe this will solve the problem

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -45,6 +45,7 @@ intellijPlatform {
     version = provider { project.version.toString() }
     ideaVersion {
       sinceBuild = "242"
+      untilBuild = "242.*"
     }
```
Comment (aSemy) [inline build.gradle.kts]: https://github.com/aSemy/ij-problem-ignorer/blob/v0.0.4/build.gradle.kts#L44-L47

This is the same config I had in v0.0.4 though :(

Comment (CrazyBoy49z) [reply build.gradle.kts]: I’ll be watching over the weekend, can you tell me how I can build the plugin locally?

Comment (aSemy) [reply build.gradle.kts]: This repo uses the official IJ Gradle Plugin, so all the info should be in the docs :)

https://plugins.jetbrains.com/docs/intellij/tools-intellij-platform-gradle-plugin.html

Comment (CrazyBoy49z) [reply build.gradle.kts]: > This repo uses the official IJ Gradle Plugin, so all the info should be in the docs :)
> 
> https://plugins.jetbrains.com/docs/intellij/tools-intellij-platform-gradle-plugin.html

FIxed

Comment (CrazyBoy49z) [reply build.gradle.kts]: image

**Comment by aSemy** [comment]
See https://github.com/aSemy/ij-problem-ignorer/issues/1#issuecomment-2383717404

**Pull request closed without merging**